pandas/core/groupby/groupby.py

```diff
@@ -272,7 +282,7 @@ class providing the base-class of operations.
     each group together into a Series, including setting the index as
     appropriate:
 
-    >>> g1.apply(lambda x: x.C.max() - x.B.min())
+    >>> g1[['B', 'C']].apply(lambda x: x.C.max() - x.B.min())
```

Comment:
Expressions like these (`[['B', 'C']]`) will be redundant in v3.0, so if we instead do `g1.apply(lambda x: x.C.max() - x.B.min(), include_groups=False)` they will be easier to find and remove in v3.0 and IMO bracketed expressions are a bit ugly...

Just a suggestion, you decide :-)

Reply:
I don't have any issues with the aesthetics, but completely agree on making it easier to cleanup. Thanks for catching this.